lib/OpenQA/Task/Job/FinalizeResults.pm

```diff
@@ -49,15 +49,16 @@ sub _finalize_results {
     return if $carried_over;
     my $key = 'job_done_hook_' . $openqa_job->result;
     if (my $hook = $ENV{'OPENQA_' . uc $key} // $app->config->{hooks}->{lc $key}) {
-        my $ret = _done_hook_new_issue($openqa_job, $hook);
+        my $timeout      = $ENV{'OPENQA_JOB_DONE_HOOK_TIMEOUT'}      // '5m';
+        my $kill_timeout = $ENV{'OPENQA_JOB_DONE_HOOK_KILL_TIMEOUT'} // '30s';
```

Comment:
There is no need to quote the hash keys (`$ENV{OPENQA_JOB_DONE_HOOK_KILL_TIMEOUT}`).

Reply:
It's actually inconsistent with the other uses of the same env var in the test.